### ISSUE TITLE
Snapping: Resizing blocks in relation to other blocks

### ISSUE THREAD

**Issue by swissspidy** [ampproject/amp-wp#2994]
See #2875 for details.

Example: when resizing a block it should be easy to make it the same size as other blocks, as indicated by snapping indicators.

**Comment by swissspidy** [comment]
See #2992 for screencast.

**Comment by swissspidy** [comment]
**Testing Instructions**

1. Create new story
1. Add a new block (so there should be 2 now)
1. Add background colors to both blocks
1. Resize one of the blocks
1. For each of the block's sides (left, right, top, bottom) and centers (vertical and horizontal), verify that red snapping guidelines are shown in the following scenarios:  
    1. When resizing the block so it aligns with either side of other block
    1. When resizing the block so it aligns with the center (vertical and horizontal) of the other block.
1. Verify that this also happens when there are N blocks on a page.

Basically, snapping guidelines should be shown to allow easily aligning blocks to the sides and center of the other. No actual _snapping_ is done.

Next, verify that no guidelines are shown when resizing and at the same time holding down the alt/option key.

**Comment by csossi** [comment]
Verified in QA
